osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -14,51 +16,115 @@ namespace osu.Game.Rulesets.Osu.Difficulty.Skills
     /// </summary>
     public class Aim : OsuStrainSkill
     {
-        private const double angle_bonus_begin = Math.PI / 3;
-        private const double timing_threshold = 107;
-
         public Aim(Mod[] mods)
             : base(mods)
         {
         }
 
-        protected override double SkillMultiplier => 26.25;
+        protected override int HistoryLength => 2;
+
+        protected override double SkillMultiplier => 24.75;
         protected override double StrainDecayBase => 0.15;
 
+        private const double wide_angle_multiplier = 1.0;
+        private const double acute_angle_multiplier = 1.0;
+        private const double rhythm_variance_multiplier = 1.0;
```

Comment:
Yes. I put it there in case people want to adjust it on their own to try balancing for themselves. 1.0 worked for me but it may req buffs or nerfs in future.